core/src/main/java/nl/nn/adapterframework/stream/json/JsonWriter.java

```diff
@@ -138,7 +138,7 @@ public void primitive(Object value) throws SaxException {
 		try {
 			writeSeparatingComma(false);
 			if (value instanceof String) {
-				writer.write("\""+value+"\"");
+				writer.write("\""+((String)value).replaceAll("\"", "\\\\\"").replaceAll("\n", "\\\\n")+"\"");
```

Comment:
Dan moet dit naar een JsonMisc want volgens mij zit dit al op een paar plekken zo, en dat is niet zo handig..